conf/ingress.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: orsp-pub-dev-ip
+  labels:
+    app: ${REPO_NAME}
+  name: ${REPO_NAME}-ingress
+  namespace: default
```

Comment:
We don't recommend the default namespace so that the cluster could be used for other namespaced apps.

Reply:
Got it. I'll move all to an orsp namespace.